Abstract.md

```diff
@@ -1,38 +1,40 @@
 # Abstract for conferences
 
-Title proposal: 
+Title proposal:
 
-End-to-end workflow for LC-MS/MS pre-processing and analysis using 
-open-source *xcms* software and the *RforMassSpectrometry* environment.
+End-to-end workflow for reproducible analysis of LC-MS/MS data in R using
+*xcms* and packages from the *RforMassSpectrometry* initiative.
 
-OR 
+OR
 
-Streamlining LC-MS/MS Data Analysis with Open-Source *xcms* and 
+Streamlining LC-MS/MS Data Analysis in R with Open-Source *xcms* and
 *RforMassSpectrometry*: An End-to-End Workflow
 
 
-Despite untargeted LC-MS/MS data being a powerful approach in investigating 
-metabolomics, a significant challenge in the field lies in the absence of a 
-reproducible, scalable and stable infrastructure for its analysis. While various 
```

Comment:
I think we should less focus on a "infrastructure" (that's something we do with RforMassSpectrometry) but here focus on an actual reproducible workflow.